mantis-runtime-loader/src/main/java/io/mantisrx/runtime/loader/config/WorkerConfiguration.java

```diff
@@ -155,6 +155,18 @@ default Time getHeartbeatInterval() {
     @DefaultNull
     File getLocalStorageDir();
 
+    @Config("mantis.taskexecutor.hardware.cpu-cores")
```

Comment:
why are these moving to worker config vs the current reporting structure from a util/hardware layer? It doesn't seem intuitive to me these metadata should be part of the "configuration" object.

Reply:
We have had bugs in the past where the machine would have a profile that does not match the requested resources resulting in scheduling issues on the control plane.  Hence, this change is expected to address that.